source/manual/alerts/signon-api-user-token-expires-soon.html.md

```diff
@@ -58,6 +58,6 @@ Once you've confirmed that the new token can be used, go back to [Signon] and re
 
 ## Special Cases
 
-If the tokens are for `New London: Trade Tariff Admin (PaaS) token for New London: Trade Tariff Backend (PaaS)` see [Trade Tariff Admin on the Wiki](https://gov-uk.atlassian.net/wiki/spaces/PLOPS/pages/3155099649/Trade+Tariff+Admin)
+If the tokens are for `New London: Trade Tariff Admin token for New London: Trade Tariff Backend` see [Trade Tariff Admin on the Wiki](https://gov-uk.atlassian.net/wiki/spaces/PLOPS/pages/3155099649/Trade+Tariff+Admin)
```

Comment:
This token is still called "New London: Trade Tariff Admin (PaaS)" as per https://signon.publishing.service.gov.uk/api_users, so we should leave this unchanged. (Though we can remove the weird duplication inside the backticks)